File: Exercicios_guanaraba/Com_Dart/bin/ex013_Reajuste_Salarial.dart
//Faça um algoritmo que leia o salário de um funcionário e mostre seu novo salário, com 15%
//de aumento.
import 'dart:io';

void main(){

  print ('Insira o seu salário: ');
  double salario = double.parse(stdin.readLineSync());
  double novosal= salario + (salario * 15 /100);
  print ('Seu salário com 15% de aumento foi para: $novosal');


}

File: Exercicios_guanaraba/Com_Ruby/exercicios/ex013_Reajuste_Salarial.rb
#Faça um algoritmo que leia o salário de um funcionário e mostre seu novo salário, com 15% de aumento.

puts "Insira seu salario: "
salario = gets.chomp.to_f
novosal = salario +(salario * 15 / 100)
puts "Seu salario com 15% de aumento foi para : #{novosal}"